src/packet/single.rs

```diff
@@ -127,6 +127,7 @@ pub fn body_parser(ver: Version, tag: Tag, body: &[u8]) -> Result<Packet> {
         }
         Tag::ModDetectionCode => ModDetectionCode::from_slice(ver, body).map(Into::into),
         Tag::Padding => Padding::from_slice(ver, body).map(Into::into),
+        Tag::Other(20) => unimplemented_err!("GnuPG AEAD is unsupported and not part of OpenGPG"),
```

Comment:
```suggestion
        Tag::Other(20) => unimplemented_err!("GnuPG-prorietary 'OCB Encrypted Data Packet' is unsupported"),
```

Reply:
Thanks for adding a message, this is useful! I suggested a minor change, above, to reflect the exact name of packet type 20 in the draft text of GnuPG's format.

Reply:
Oops, thanks for catching my typo :relaxed: